app/views/comments/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Comments#create</h1>
```

Comment:
- [ ] There should be a form in the `views/comments/new.html.erb` file which corresponds to the `new` action in _comments controller_, that accepts input for creating comments. You currently don't have this required _view_ file. Kindly ensure to improve upon the same.

Reply:
@rloterh  Thank you, I have implemented the required changes.